api/geth_backend.go

```diff
@@ -1751,7 +1751,12 @@ func (b *GethStatusBackend) loadNodeConfig(inputNodeCfg *params.NodeConfig) erro
 	conf.Version = params.Version
 	conf.RootDataDir = b.rootDataDir
 	conf.DataDir = filepath.Join(b.rootDataDir, conf.DataDir)
-	conf.ShhextConfig.BackupDisabledDataDir = filepath.Join(b.rootDataDir, conf.ShhextConfig.BackupDisabledDataDir)
```

Comment:
How does this approach differ from handling `KeyStoreDir`? Do we need to do the same check for `KeyStoreDir`?